src/ui/components/Icon/styles.scss

```diff
@@ -174,3 +178,90 @@
 .Icon-x-mark {
   @include icon($name: "x-mark");
 }
+
+.Icon-permission-bookmarks {
+  @include permission-icon($name: "bookmarks");
+}
+
+.Icon-permission-browserSettings,
+.Icon-permission-proxy {
+  @include permission-icon($name: "browser");
+}
+
+.Icon-permission-browsingData {
+  @include permission-icon($name: "info");
+}
+
+.Icon-permission-clipboardRead,
+.Icon-permission-clipboardWrite {
+  @include permission-icon($name: "clipboard");
+}
+
+.Icon-permission-devtools {
+  @include permission-icon($name: "dev");
+}
+
+.Icon-permission-downloads,
+.Icon-permission-downloads-open {
+  @include permission-icon($name: "downloads");
+}
+
+.Icon-permission-find {
+  @include permission-icon($name: "search");
+}
+
+.Icon-permission-geolocation {
+  @include permission-icon($name: "geolocation");
+}
+
+.Icon-permission-history {
+  @include permission-icon($name: "history");
+}
+
+.Icon-permission-management {
+  @include permission-icon($name: "settings");
+}
+
+.Icon-permission-nativeMessaging {
+  @include permission-icon($name: "messages");
+}
+
+.Icon-permission-notifications {
+  @include permission-icon($name: "notifications");
+}
+
+.Icon-permission-pkcs11 {
+  @include permission-icon($name: "cryptographic");
+}
+
+.Icon-permission-privacy,
+.Icon-permission-hostPermission {
+  @include permission-icon($name: "privacy");
+}
+
+.Icon-permission-sessions,
+.Icon-permission-tabs,
+.Icon-permission-tabHide {
+  @include permission-icon($name: "tabs");
+}
+
+.Icon-permission-topSites {
+  @include permission-icon($name: "top-sites");
+}
+
+.Icon-permission-unlimitedStorage {
+  @include permission-icon($name: "storage");
+}
+
+.Icon-permission-webNavigation {
+  @include permission-icon($name: "navigation");
+}
+
+.PermissionCard-learn-more.Button--neutral:link.Button--puffy {
```

Comment:
I tried this, although adding a new toggle for `externalBlack` seemed less than ideal because one ends up with two toggles that do the same thing (create an `Icon`), which requires extra code to untangle. Instead I added a property for `blackIcon` which simply controls the `name` used for the `Icon`. Note that this still didn't entirely deal with the specificity problem I was having with the css. I still needed to define a style that includes `.Button--neutral:link.Button--puffy` to make it work. :(